File: seed/java-sdk/examples/inline-file-properties/src/main/java/com/snippets/Example17.java
package com.snippets;

import com.seed.examples.SeedExamplesClient;
import com.seed.examples.resources.service.requests.GetMetadataRequest;
import java.util.Arrays;
import java.util.Optional;

public class Example17 {
    public static void main(String[] args) {
        SeedExamplesClient client = SeedExamplesClient
            .builder()
            .token("<token>")
            .url("https://api.fern.com")
            .build();

        client.service().getMetadata(
            GetMetadataRequest
                .builder()
                .xApiVersion("0.0.1")
                .tag(
                    Arrays.asList(Optional.of("development"))
                )
                .shallow(false)
                .build()
        );
    }
}

File: seed/csharp-sdk/examples/readme-config/Snippets/Example17.cs
using SeedExamples;

public partial class Examples
{
    public async Task Example17() {
        var client = new SeedExamplesClient(
            token: "<token>",
            clientOptions: new ClientOptions {
                BaseUrl = "https://api.fern.com"
            }
        );

        await client.Service.GetMetadataAsync(
            new GetMetadataRequest {
                Shallow = false,
                Tag = new List<string>(){
                    "development",
                }
                ,
                XApiVersion = "0.0.1"
            }
        );
    }

}
